src/lib/Scenes/Artwork/Components/__tests__/RequestConditionReport-analytics-tests.tsx

```diff
@@ -0,0 +1,85 @@
+import { Button } from "@artsy/palette"
+import { mount } from "enzyme"
+import Event from "lib/NativeModules/Events"
+import { flushPromiseQueue } from "lib/tests/flushPromiseQueue"
+import { mockTracking } from "lib/tests/mockTracking"
+import React from "react"
+
+jest.unmock("react-tracking")
+
+import { RequestConditionReport } from "../RequestConditionReport"
+
+jest.mock("lib/NativeModules/Events", () => ({ postEvent: jest.fn() }))
+
+const artwork = {
```

Comment:
I imagine that this test artwork could not be identity to the one in the above spec https://github.com/artsy/emission/pull/2050/files#diff-a27d3021ebaac1cf475edd4295f15fedR79

which would be more consistent, and also have the same type safety benefits.

Reply:
👍 Good point, added to this and `RequestConditionReport-tests.tsx"